### PR TITLE
Jon/smol-4.8

### PR DESCRIPTION
**Pull request by Jon-edge**
image
image
image
image
image
image
image
image

image
image
image
image
image
image

### CHANGELOG

Does this branch warrant an entry to the CHANGELOG?

- [x] Yes
- [ ] No

### Dependencies

 none

### Requirements

If you have made **any** visual changes to the GUI. Make sure you have:

- [x] Tested on iOS device
- [x] Tested on Android device
- [ ] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207225096190322

### CODE REVIEW

File: src/components/ui4/RowUi4.tsx [hunk 1]
```diff
@@ -42,13 +44,21 @@ export const RowUi4 = (props: Props) => {
...
 
   const margin = sidesToMargin(mapSides(fixSides(marginRem, 0.5), theme.rem))
 
   const numberOfLines = textHeights[maximumHeight]
 
+  // TODO: Merge styles.containerTemp into styles.container permanently.
+  // Explicitly setting the container style here to avoid unit test snapshot
+  // diffs.
+  const containerStyle: StyleProp<ViewStyle> = React.useMemo(
+    () => (shrinkBody ? [styles.container, margin, styles.containerTemp] : [styles.container, margin]),
```
Comment (paullinator) [inline src/components/ui4/RowUi4.tsx:58]: should be explicit boolean check `shrinkBody === true`

File: src/components/scenes/CreateWalletSelectCryptoScene.tsx [hunk 2]
```diff
@@ -288,7 +288,7 @@ const CreateWalletSelectCryptoComponent = (props: Props) => {
     () => (
       <Fade noFadeIn={defaultSelection.length > 0} visible={selectedItems.size > 0} duration={300}>
         <View style={styles.bottomButton}>
-          <ButtonUi4 label={lstrings.string_next_capitalized} onPress={handleNextPress} />
+          <ButtonUi4 label={lstrings.string_next_capitalized} testID="createWalletCryptoNext" onPress={handleNextPress} />
```
Comment (paullinator) [inline src/components/scenes/CreateWalletSelectCryptoScene.tsx:291]: Did you test that this works? Usually you need to force the parent container to not takeover taps by putting
`accessible={false}` on the parent.

Or the problem is due to absolute positioning.

**Review by paullinator — CHANGES_REQUESTED** [review]

File: src/components/scenes/CreateWalletSelectCryptoScene.tsx (hunk 2 above)
Comment (Jon-edge) [reply src/components/scenes/CreateWalletSelectCryptoScene.tsx:291]: `accessible={false}` had no impact to the behavior. There was no Maestro tappability issues through this View when disabling the fade animation for Maestro builds.

Kind of related: `isMaestro()` itself is broken - does not return true for Maestro builds.

Comment (Jon-edge) [reply src/components/scenes/CreateWalletSelectCryptoScene.tsx]: Use of `EdgeAnim` helps, but only to a degree. Nesting buttons within a hard-coded styled `View` as done before doesn't work no matter what. `ButtonsView` flavors such as `SceneButtons` works perfectly and makes the spacing consistent.

**Review by paullinator — APPROVED** [review]

**Pull request merged**